### ISSUE TITLE
IcedSourceShow or IcedSourcePopupShow not working

### ISSUE THREAD

**Issue by glyh** [liquidz/vim-iced#461]
Here's the error message I got:
```
Error detected while processing function <SNR>88__worker[12]..function <SNR>88__worker[12]..<SNR>87__publish[22]..<SNR>87__invo
ke_callback[6]..<lambda>1723[1]..<SNR>115___extract_source[8]..53:                                                             
line    1:                                                                                                                     
E484: Can't open file zipfile:///home/lyh/.m2/repository/org/clojure/clojure/1.11.1/clojure-1.11.1.jar::clojure/core.clj 
```

I'm using neovim.

**Comment by liquidz** [comment]
@glyh Thank you for your reporting!
I'll have a look later.

**Comment by liquidz** [comment]
@glyh I've tried to fix it in [bugfix/neovim_show_source](https://github.com/liquidz/vim-iced/tree/bugfix/neovim_show_source) branch.
Could you try?

**Comment by glyh** [comment]
Yeah it's working, thank for being very responsive.

**Comment by liquidz** [comment]
@glyh Thanks for your confirmation!
Just released v3.14.3240.
